### PR TITLE
Trainee homepage

### PR DESCRIPTION
**Pull request by berlyseria**
Working on the functionality of the trainee homepage

### CODE REVIEW

**Comment by yipingbenjaminliu** [comment]
@berlyseria , 
try and abstract the javascript code that's duplicated in bible_reading into a js file and use it for both instead of duplicating it here
the css code should also be siting by itself instead of in the index.html

File: ap/bible_tracker/static/bible_tracker/js/bible_reading.js [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+    if (is_trainee && (now > WedofNextWeek || now < lastDayofWeek.clone().add(20, "minutes"))) {
+        $("#finalize").prop("disabled", true);
+    }
+    //save button is disabled for more than this week"s Bible reading
+    if (is_trainee && lastDayofWeek >= now.clone().add(6, "days")) {
+        $("#save").prop("disabled", true);
+    }
+}
+
+function modify_boxes(classname) {
```
[the remark this answers is not in the document]
Comment (davidhtien) [reply ap/bible_tracker/static/bible_tracker/js/bible_reading.js]: Would be good to change this to camelCase

File: ap/bible_tracker/static/bible_tracker/js/bible_reading.js [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+    for (var i = 0; i < 7; i++) {
+        $("#day-" + i).html(currentDayofWeek.format("ddd - MMM D"));
+        currentDayofWeek.add(1, "days");
+    }
+    //finalize is disabled on Wednesday of next week (with HH:mm of 00:00) or before week is over (Sunday midnight+20 minutes)
+    if (is_trainee && (now > WedofNextWeek || now < lastDayofWeek.clone().add(20, "minutes"))) {
+        $("#finalize").prop("disabled", true);
+    }
+    //save button is disabled for more than this week"s Bible reading
+    if (is_trainee && lastDayofWeek >= now.clone().add(6, "days")) {
```
[the remark this answers is not in the document]
Comment (davidhtien) [reply ap/bible_tracker/static/bible_tracker/js/bible_reading.js]: where does `is_trainee` come from? Can it be made camelCase?

File: ap/bible_tracker/static/bible_tracker/js/bible_reading.js [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+}
+
+function updateProgressBar(selector, data) {
+    $(selector).css("width", data + "%");
+    $(selector).html(data + "%");
+    $(selector).attr("aria-valuenow", data);
+}
+
+function unfinalizeStatus() {
+    var week_id = $("#week_select").find(":selected").attr("id");
```
[the remark this answers is not in the document]
Comment (davidhtien) [reply ap/bible_tracker/static/bible_tracker/js/bible_reading.js]: camelCase

File: libraries/bootflat-ftta/bootflat/scss/bootflat/_table.scss [hunk 4]
```diff
@@ -128,8 +131,9 @@ $table-text-color:                darken($brown, 50%) !default;
       > tr {
         > th {
           color: $grey;
-          border-top: 1.5px solid $grey-light;
-          border-left: 1.5px solid $grey-light;
+          border: 1.5px solid $grey-light;
+          // border-top: 1.5px solid $grey-light;
+          // border-left: 1.5px solid $grey-light;
```
Comment (davidhtien) [inline libraries/bootflat-ftta/bootflat/scss/bootflat/_table.scss]: This seems interesting... what other views does it affect? Just want to make sure you checked with the designers about this

**Pull request merged**